frontend/src/components/smart/PvPArena.vue

```diff
@@ -422,6 +484,19 @@ export default {
           return formatDuelResult(duel.returnValues);
         });
 
+        const kickedEvents = await this.getKickedEvents(this.contracts());
+
+        if (kickedEvents.length && !this.isCharacterInArena) {
+          const formattedResult = formatCharacterKickedEvent(kickedEvents[kickedEvents.length - 1].returnValues);
+
+          if (+localStorage.getItem(`${getCharacterNameFromSeed(this.currentCharacterId)}: lastKickedTime`) < +formattedResult.timestamp) {
```

Comment:
Note that you can also just `JSON.stringify` and `JSON.parse` so that you don't have to create these keys. Up to you if you think the change is worth making though